tests/http1_tests.py

```diff
@@ -272,13 +273,11 @@ def http1_ping(sport, cport):
         ]
     }
 
-    server = TestServer(server_port=sport,
-                        client_port=cport,
-                        tests=TEST)
+    server = wait_server_starting(sport, cport, TEST)
```

Comment:
This problem would go away entirely if the test started the server first, before routers, with `TestServer(server_port=0, ...` and then queried the server for an actual port when starting the routers. But if the approach of the PR works reliably, then I see no reason to do yet another change to it.

Reply:
The tests bring up and bring down the TestServer (in some cases 2 test servers). The TestServer(s) seem to shut down successfully but the port is not released on time before the next test comes along and starts the next TestServer on the same port(s). This fix will wait for the few seconds until the port is released. Could take more than a few seconds for the port to be released but that is something I have not seen yet.